sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -100,13 +112,7 @@ internal class RadarApiClient(
                 if (status == RadarStatus.SUCCESS) {
                     Radar.flushLogs()
                 }
```

Comment:
@jkao, when I re-added the `flushLogs()` call in [the iOS version](https://github.com/radarlabs/radar-sdk-ios/pull/127), I didn't add a check for a successful status. Should I?

Reply:
it may actually be better that it lives without a status check. we already have a "guard" in the sense that we don't flush the logs if the buffer is empty, so doesn't feel like we need this. it may be more informative if we do log http errors in any case.

Reply:
I'd prefer not to change this, since it is already thoroughly dogfooded with the current behavior. Main point of this PR is to add the new feature, not modify existing features.